stdci.yaml

```diff
@@ -1,5 +1,5 @@
 sub-stages:
-  - k8s-multus-1.12.2-release
```

Comment:
Should we delete the cluster/k8s-multus-1.12.2 folder?

Reply:
I don't have any strong feelings about that we can leave it if any one want to validate something in a older version)